### PR TITLE
Fixes violations of and reenables rubocop Lint/EndAlignment.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Fixes generated manually. Note also that our code seems to surface a bug in rubocop, as rubocop fails to find the second failure in each of these file.

Might mess around with this, trying to produce a simple example with this failure (trivial examples do not surface the bug), reporting an issue if I am able to do so.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Ping.

**Comment by mehalshah** [comment]
Sorry - looking now

**Comment by mehalshah** [comment]
LGTM

File: cookbooks/cdo-varnish/libraries/helpers.rb [hunk 1]
```diff
@@ -146,13 +146,13 @@ def process_request(behavior, _)
...
-    else
-      cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
-  end
+         when 'all'
+           '# Allow all request cookies.'
+         when 'none'
+           'cookie.filter_except("NO_CACHE");'
+         else
+           cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
+         end
```
Comment (joshlory) [inline cookbooks/cdo-varnish/libraries/helpers.rb]: For `Style/AlignParameters` we've preferred fixed indentation.  I wonder if we should use parens here keep the `case` statement at a normal indent?

``` ruby
out << (
  case cookies
  when 'all'
    '# Allow all request cookies.'
  when 'none'
    'cookie.filter_except("NO_CACHE");'
  else
    cookies.map{ |c| extract_cookie(c)}.join + "cookie.filter_except(\"#{cookies.join(',')}\");"
  end
)
```

We've been using this pattern a lot in React ([example](https://github.com/code-dot-org/code-dot-org/blob/1f6b5050c3fd8700b3618a7b9f45286534e3810f/apps/src/templates/instructions/MarkdownInstructions.jsx#L94-L103)).

Comment (ashercodeorg) [inline cookbooks/cdo-varnish/libraries/helpers.rb]: Done (also removed unnecessary assignment of `out` to empty string).

**Pull request merged**